containers/server-image/Dockerfile

```
@@ -7,12 +7,14 @@ FROM $INIT_IMAGE
 ARG PRODUCT_PATTERN_PREFIX="patterns-uyuni"
 
 # Extra packages can be added via project configuration
-ARG EXTRAPACKAGES
+ARG EXTRAPACKAGES=shim%20nss-pam-ldapd
```

```suggestion
ARG EXTRAPACKAGES=nss-pam-ldapd
```
`shim` should not be in the defaults: it is only for some architectures